src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,136 @@
+package duke;
+
+import java.time.LocalDate;
```

Comment:
Here the import statements can be separated to determine the dependencies more easily

Reply:
Hmm, seprarting the import statments seems to violate the checkstyle 🤔